### DISCUSSION TITLE
Saving decoded messages - error message "not enough fields"

### DISCUSSION THREAD

**Discussion by jeagh** [szpajder/dumphfdl#44] [Q&A]
I'm having great success with dumphfdl and a poor antenna. Now I'd like to plot the aircraft's position on a map. I've followed the directions to another user's question but I get the following error message:

$ dumphfdl --soapysdr driver=sdrplay --sample-rate 250000 21928 21931 21934 21937 21949 21955 21982 21990 21995 21997 --output decoded:basestation:file=/home/phil/positions.log,rotate=daily

The error message reads:

dumphfdl 1.6.1 (libacars 2.2.0)
Could not parse output specifier 'decoded:basestation:file=/home/phil/positions.log,rotate=daily': not enough fields

I have complied in all of the options except etsy.

**Comment by dg9bja** [comment]
Please put the frequencies to the end of the command.

**Reply by szpajder** [reply]
This is not required. Channel frequencies can be anywhere on the command line.

**Comment by szpajder** [comment]
The correct syntax is:
decoded:basestation:file:path=/home/phil/positions.log,rotate=daily .

**Reply by jeagh** [reply]
Thank you, problem solved.

Rather than save the decodes to a file I'm now using VSR to display the aircraft in real time. The only problem that I have now is my antenna is too poor to track aircraft but they do show up on the map.
